src/RoutingList/RoutingList.js

```diff
@@ -0,0 +1,162 @@
+import {
+  useCallback,
+  useState,
+} from 'react';
+import {
+  FormattedMessage,
+  useIntl,
+} from 'react-intl';
+import {
+  useHistory,
+  useParams,
+} from 'react-router';
+
+import { useShowCallout } from '@folio/stripes-acq-components';
+import {
+  Accordion,
+  AccordionSet,
+  AccordionStatus,
+  Button,
+  Col,
+  ConfirmationModal,
+  ExpandAllButton,
+  Icon,
+  KeyValue,
+  LoadingView,
+  Pane,
+  Row,
+} from '@folio/stripes/components';
+import { AppIcon, IfPermission } from '@folio/stripes/core';
+import { ViewMetaData } from '@folio/stripes/smart-components';
+
+import {
+  useRoutingListById,
+  useRoutingListMutation,
+} from './hooks';
+import { RoutingListUsers } from './RoutingListUsers';
+
+export const RoutingList = () => {
+  const history = useHistory();
+  const showCallout = useShowCallout();
+  const intl = useIntl();
+  const { id } = useParams();
+
+  const [confirmDelete, setConfirmDelete] = useState(false);
```

Comment:
`confirmDelete` is perceived as a function name, not a boolean value.

Reply:
there is hook useToggle